uber/site_sections/budget.py

```diff
@@ -114,20 +114,23 @@ def generate_promo_codes(
             self,
             session,
             message='',
-            is_single_promo_code=0,
+            is_single_promo_code=1,
             count=1,
             use_words=False,
+            length=9,
```

Comment:
A lot of these values are just repeated 'defaults' from the PromoCode class -- we definitely want to avoid this if we can.

For length and segment length specifically, we define the defaults here, down in the block on line 177, down in the 'successful return,' AND in the generation code at https://github.com/magfest/ubersystem/blob/master/uber/models.py#L2446.

My recommendation is to modify this function to pass a falsey value to generate_random_code, and then in that function do a simple `length = length if length else 9` check (possibly with a comment that explains why). That way we won't get any value mismatches later. It also saves us from actually having to pass any values for length or segment_length to the template (though I guess we'd want placeholder or help text telling the admin what the default is...)